Altis_Life.Altis/core/shops/fn_vehicleShopBuy.sqf

```diff
@@ -44,9 +44,17 @@ switch (playerSide) do {
 };
 
  if (_mode) then {
-     _purchasePrice = round(_initalPrice * _buyMultiplier);
+     if ((_customPriceArray select 0) isEqualTo -1) then {
```

Comment:
Let's make a fallback case if the end user didn't add the needed array (doing this for other things currently, stops all the problems appearing on Discord/forums)